IntelliJ guide, analysis chapter (#518)
windup/windup-documentation: docs/topics/intellij-idea-plugin-reviewing-issues.adoc

// Module included in the following assemblies:
//
// * docs/intellij-idea-plugin-guide/master.adoc

[id="intellij-idea-plugin-reviewing-issues_{context}"]
= Reviewing issues

> You can use the {ProductShortName} plugin icons to prioritize issues based on their severity. You can  see which issues have a _Quick Fix_ automatic code replacement and which do not.

Reviewer: Seems like _Quick Fix_ means "automatic code replacement". If so, then define it here, and just use "Quick Fix" subsequently, without "automatic code replacement".

Author: Done.

The results of an analysis are displayed in a directory format, showing the _hints_ and _classifications_ for each application analyzed.

> A hint is a read-only snippet of code that contains a single issue that either must or should be addressed before an application can modernized or migrated. Often a suggested Quick Fix code replacement is suggested. You can accept or ignore any suggested Quick Fixes.

Reviewer: Lot of passive voice in this sentence.

```suggestion
A _hint_ is a read-only snippet of code that contains a single issue that you should or must address before you can  modernize or migrate an application. Often the plugin suggests a Quick Fix, which you can either accept or ignore.
```

Author: I've incorporated your recommendations except for in the last sentence. You suggest using an active verb with the system, which violates the rule against anthropomorphism.

A classification is a file that has an issue but does not have any suggested Quick Fixes. You can edit a classification.

.Procedure

> . In the {ProjectName} view, select a run configuration directory on the left pane.

Reviewer: on -> in

Author: Done.

Reviewer: LGTM

+

> The *Report* and *Results* icons are displayed.

Reviewer: Are they icons or buttons?

Author: icons.

Reviewer: LGTM

. Click the *Results* icon.
+
The modules and applications of the run configuration are displayed. Below each application, the *Hints* and *Classifications* icons are displayed.

. Prioritize issues based on the following icons, which are displayed next to each hint:

> ** image:intellij-idea-mandatory.png[Mandatory] : The issue must be fixed for a successful migration.

Reviewer: passive voice here an the next in the item.

Author: Fixed.

Reviewer: lgtm

** image:intellij-idea-potential.png[Warning] : The issue might need to be addressed during migration.

> . Optional: To learn more about a hint, right-click on it and select *Show More Details*.

Reviewer: Style guideline: You click something, you don't click _on_ something.

So "right-click it". Search for other instances of "click on" as well.

Author: Done.

Reviewer: lgtm
